tables/tables-list.csv

```diff
@@ -1,38 +1,38 @@
 TABLE_NAME,CONTAINS_TAXYR,NUM_BUCKETS
 AASYSJUR,FALSE,1
-ADDN,TRUE,10
+ADDN,TRUE,1
 ADDRINDX,TRUE,1
-APRVAL,TRUE,20
-ASMT_ALL,TRUE,30
-ASMT_HIST,TRUE,30
+APRVAL,TRUE,1
+ASMT_ALL,TRUE,20

```

Comment:
**[Question, non-blocking]** Given that `cur` is supposed to be an enumerated type with at most 3 values, does it still make sense to set `NUM_BUCKETS=20`?

Reply:
The bucketing here is actually by `parid`, `cur` is just a field to internally sort the Parquet once it's bucketed.